src/components/Cards/video/index.jsx

```diff
@@ -0,0 +1,29 @@
+/** @format */
+
+import React from "react";
+import "./videoCard.css";
+import { Link } from "react-router-dom";
+import { ReactComponent as MenuIcon } from "../../../assets/ElipsesIcon.svg";
```

Comment:
Use absolute path. Remove  .. /../

Reply:
Don't know about it will learn and then implement it.